### PR TITLE
Get cpu usages

### PR DESCRIPTION
**Pull request by minsulee2**
Add cpu usage using os package.

### CODE REVIEW

**Comment by cshcomcom** [comment]
Thanks! LGTM

**Review by cshcomcom — APPROVED** [review]
Thanks! LGTM

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

File: p2p/server.js [hunk 1]
```diff
@@ -184,6 +184,24 @@ class P2pServer {
     }
   }
 
+  getCpuUsage() {
+    const cpuUsage = {
+      free: 0,
+      usage: 0,
```
Comment (platfowner) [inline p2p/server.js:190]: minor: what about not setting this value as it will be overwritten below anyway?

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Thanks!

File: p2p/server.js (hunk 1 above)
Comment (minsulee2) [reply p2p/server.js]: Done!!

**Comment by minsulee2** [comment]
Thanks for the fast reviews!! merging now :)

**Pull request merged**